src/main/java/com/palantir/gradle/gitversion/VersionDetails.java

```diff
@@ -97,6 +110,10 @@ private String expensiveComputeRawDescription() {
     }
 
     private boolean isRepoEmpty() {
+        if (isWorktree()) {
+            return false; // not implemented for worktrees yet
```

Comment:
same here, what causes work trees to be different

Reply:
The jgit implementation of this method is poorly implemented, and will throw an exception (and then return `true`) in case we're in a worktree directory. We could perhaps try to come up with a native command for `isRepoEmpty`, but the risk is that native `git` isn't available in the environment, so I took the safe route here for now.